app/controllers/application.js

```diff
@@ -8,44 +10,53 @@ export default Ember.Controller.extend({
     showUserOptions: false,
     search: Ember.computed.oneWay('controllers.search.q'),
 
-    stepFlash: function() {
+    stepFlash() {
         this.set('flashError', this.get('nextFlashError'));
         this.set('nextFlashError', null);
     },
 
-    aboutToTransition: function() {
+    aboutToTransition() {
         Ember.$(document).trigger('mousedown');
     },
 
-    resetDropdownOption: function(controller, option) {
-        controller.set(option, !controller.get(option));
-        if (controller.get(option)) {
-            Ember.$(document).on('mousedown.useroptions', function(e) {
+    // don't use this from other controllers..
+    resetDropdownOption(controller, option) {
+        this.set(option, !this.get(option));
+        if (this.get(option)) {
```

Comment:
How come this changed from `controller` to `this`? I'm not 100% sure why it was that way in the first place, but may be a spot of differences